src/Route/api_v1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use GoApptiv\FileManagement\src\Con;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use GoApptiv\FileManagement\Controller\FileVariantController;
+
+Route::prefix('api')->group(function () {
+    Route::put('variant-callback', [FileVariantController::class, 'updateVariantDetails']);
```

Comment:
```suggestion
    Route::post('variant-callback', [FileVariantController::class, 'updateVariantDetails']);
```

pub sub only triggers post method